lighttpd/lighttpd.conf.template

```diff
@@ -93,5 +94,18 @@ server.groupname = "%(GROUP)s"
 
 server.upload-dirs = ( "/tmp" )
 
+websocket.server = ( "/bytecount" =>
+                    (
+                     "host" => "127.0.0.1",
+                     "port" => "80"
+                    )
+                   )
+
+# @ThePletch : Timeout for websocket connections in seconds
+websocket.timeout = 20
+
+websocket.debug = 4
```

Comment:
values:
0 - disable logging
1 - log fatal messages
2 - log error messages
3 - log warning messages
4 - log debug messages

each logs anything that falls under a log level less than the current setting